tests/kernel/install_ltp.pm

```diff
@@ -184,12 +185,13 @@ sub install_from_git {
     # It is a shallow clone so 'git describe' won't work
     script_run 'git log -1 --pretty=format:"git-%h" | tee /opt/ltp_version';
 
-    assert_script_run 'make autotools';
-    assert_script_run("$configure $extra_flags", timeout => 300);
-    assert_script_run 'make -j$(getconf _NPROCESSORS_ONLN)', timeout => $timeout;
+    assert_script_run "make autotools 2>&1 | tee -a $buildlog";
```

Comment:
I'm not sure, but AFAIR with our pipe settings, we will get 0 return code even if the first command fails. So is it fine that we will loose this info? We also could consider switching to `script_run` too. WDYT?

Reply:
Depends. tests/console/consoletest_setup.pm sets `pipefail` so that should not be a problem unless the LTP tests skip this part and it has not been executed in the parent job that creates an image.